src/EditorFeatures/Core/Implementation/NavigateTo/NavigateToItemProvider.cs

```diff
@@ -74,5 +71,30 @@ public void StartSearch(INavigateToCallback callback, string searchValue)
 
             searcher.Search();
         }
+
+        private bool GetSearchCurrentDocumentOption(INavigateToCallback callback)
+        {
+            try
```

Comment:
Do we have a bug tracking the removal of this hack? (Entirely OK with the hack though.)